modules/server/src/mapping/createConnectionResolvers.ts

```diff
@@ -40,7 +42,7 @@ const createConnectionResolvers: CreateConnectionResolversFn = ({
 				}),
 			};
 		},
-		hits: resolveHits({ type, Parallel, getServerSideFilter }),
+		...(!enableAggregationMode && { hits: resolveHits({ type, Parallel, getServerSideFilter }) }),
```

Comment:
does this conditional affect the `hits` results for a regular arranger instance (i.e. without federation)?

Reply:
Yes.

Reply:
ah, second read... the negative logic threw me off.
I gather the intent was to disable the basic `hits` if the arranger node is in "Aggregation mode". didn't realise the design for the federated nodes was an "either or".
seems like I'm misunderstanding this flag?
```suggestion
		...(enableAggregationMode || { hits: resolveHits({ type, Parallel, getServerSideFilter }) }),
```